### PR TITLE
Changed links to Documentations!

### PR DESCRIPTION
**Pull request by HurricanKai**
The Links were no longer supported....

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ foreach (var stat in varusRanked.Stats)
 }
 ```
 
-You can find a list of all the available operations in [IRiotApi](RiotSharp/IRiotApi.cs).
+You can find a list of all the available operations in [TournamentApi in Documentation](http://benfradet.github.io/RiotSharp/api/RiotSharp.RiotApi.html).
```
Comment (BenFradet) [inline README.md]: very minor but could you replace

> TournamentApi in Documentation

by

> TournamentApi in the documentation

?

Same goes for the links below.

**Comment by HurricanKai** [comment]
updated!

File: README.md (hunk 1 above)
Comment (JanOuborny) [inline README.md]: As far as my English goes, I would just drop `TournamentApi `and make it like this:
`You can find a list of all the available operations in the [documentation] (...).`

This sounds more fluent for me, but I'm not a native speaker.

Comment (BenFradet) [inline README.md]: I think we should still mention the class holding the API being discussed if someone wants to dig into the code without looking at the doc.

Comment (JanOuborny) [inline README.md]: Then I would phrase it like this:
`You can find a list of all the available operations in the [documentation for the TournamentApi] (...).`

**Pull request merged**